Code/GraphMol/SynthonSpaceSearch/rascal_search_catch_tests.cpp

```diff
@@ -114,31 +114,3 @@ TEST_CASE("RASCAL Small tests") {
     }
   }
 }
-
```

Comment:
Why are you removing a test here?

Reply:
I moved it into LONG_TEST_catch_tests.cpp.